frontend/src/components/common/Pagination.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useEffect } from "react";
+import { extendTheme, chakra, ChakraProvider, Box } from "@chakra-ui/react";
+import {
+  Pagination as P,
+  usePagination,
+  PaginationNext,
+  PaginationPage,
+  PaginationPrevious,
+  PaginationContainer,
+  PaginationPageGroup,
+  PaginationSeparator,
+} from "@ajna/pagination";
+import pagination from "../../themes/components/pagination";
+
+const customTheme = {
```

Comment:
you can remove this and add `pagination` to `themes/index.ts`

Reply:
when i do that, it messes up all the buttons and how they look so I have to import pagination into the Pagination.ts file. I tried to import the theme we already had with the pagination style added to it but that gave the same result. Unless there is another way that I don't of that allows me to add pagination to themes/index.ts while not messing up the styling of the pagination on the table page!?